src/design-tokens/mixins.css

```diff
@@ -64,52 +46,20 @@
     text-decoration-color: var(
       --eds-theme-color-text-neutral-default-inverse
     ) !important;
+    background-color: var(--eds-theme-color-background-brand-primary-strong);
   }
 
   &:visited {
     color: var(--eds-theme-color-text-neutral-subtle);
     text-decoration-color: var(--eds-theme-color-text-neutral-subtle);
   }
-
-  /**
-   * Pseudo element with background color for focus state
-   * 1) Uses an absolutely-positioned pseudo element with a background color to animate it up and down
-   * 2) The pseudo element has a negative z-index to position it behind the text
-   * 3) The element is pushed down out of sight to hide it. Needs to be 101%, not 100%, to or the very top
-   *    will be visible
-   * 4) We set transform-origin: to bottom so it animates up from the bottom, not out from the middle
-   * 5) And we pull it back up on focus so it's visible
-   */
-  &::before {
-    /* 1 */
-    content: '';
-    width: 100%;
-    height: 100%;
-    position: absolute;
-    top: 0;
-    left: 0;
-    z-index: var(--eds-z-index-bottom); /* 2 */
-    background-color: var(--eds-theme-color-background-brand-primary-strong);
```

Comment:
The background-color change was moved up into the `&:focus` styles; the rest just supported the animation.